cmflib/commands/metadata/push.py

```diff
@@ -88,6 +88,8 @@ def run(self):
                 return "mlmd is successfully pushed."
             elif status_code==200 and response.json()["status"]=="exists":
                 return "Executions already exists."
+            elif status_code==200 and response.json()["status"]=="version_update":
+                return "You need to update cmf to the latest version."
             elif status_code == 404:
                 return "ERROR: cmf-server is not available."
             elif status_code == 500:
```

Comment:
Can we add something to message "Internal Server Error" to make it more informative. For example - "Internal Server Error. Check server logs."